scaling_performance/optimizing_storage.adoc

```diff
@@ -11,19 +11,167 @@
 
 toc::[]
 
-== Optimizing Storage
+This topic provides information for optimizing OpenShift storage.
+
+[[general-storage-guidelines]]
+== General Storage Guidelines
+
+The following table lists the persistent storage back-end types {product-title} can be used with.
+
+.Available storage options
+[options="header"]
```

Comment:
We can set column widths to `[cols="1,4,3",options="header"]` instead, as it renders better.